src/proto_parser.cc

```diff
@@ -164,6 +178,33 @@ LIBRARY_EXPORT int LVGetgRPCAPIVersion(int* version)
     return 0;    
 }
 
+//---------------------------------------------------------------------
+//---------------------------------------------------------------------
+LIBRARY_EXPORT int LVCreateParser(grpc_labview::LVProtoParser** parser)
+{
+    grpc_labview::InitCallbacks();
+
+    *parser = new grpc_labview::LVProtoParser();
+    return 0;
+}
+
+
+//---------------------------------------------------------------------
+//---------------------------------------------------------------------
+LIBRARY_EXPORT int LVAddParserSearchPath(grpc_labview::LVProtoParser* parser, const char* searchPath)
+{
+    parser->AddSearchPath(searchPath);
```

Comment:
We could add a nullptr check here similar to the other exports here. Same for the function below.